WebSocketClient.cs

```diff
@@ -175,8 +175,6 @@ private Task StartListenerTask()
                             break;
                         case WebSocketMessageType.Binary:
                             break;
-                        default:
-                            throw new ArgumentOutOfRangeException();
```

Comment:
Any reason this was removed?

Reply:
I believe I removed this line while I was debugging some client connection issues, but then never added it back. I'll go ahead and put it back.